tests/v3_api/common.py

```diff
@@ -24,6 +25,7 @@
 
 TEST_IMAGE = "sangeetha/mytestcontainer"
 CLUSTER_NAME = os.environ.get("RANCHER_CLUSTER_NAME", "")
+CLUSTER_NAME_2 = ""
```

Comment:
@sowmyav27 CLUSTER_NAME_2 should be read from an environment variable similar to CLUSTER_NAME

Reply:
Added CLUSTER_NAME_2 = os.environ.get("RANCHER_CLUSTER_NAME_2", "")